job-server/src/main/resources/html/js/spark-jobserver-ui.js

```diff
@@ -10,7 +10,7 @@ function getJobs() {
       $.each(jobs, function(key, job) {
         var items = [];
         items.push("<tr>");
-        items.push("<td>" + job.jobId + "</td>");
+        items.push("<td><a href='./jobs/" + job.jobId + "'>" + job.jobId + "</a> (<a href='./jobs/" + job.jobId + "/config'>C</a>)</td>");
```

Comment:
I think C is OK with me.  @noorul if you have no objections will merge this soon.

> On Jan 19, 2016, at 10:43 AM, Matthew Hokanson <notifications@github.com> wrote:
> 
> In job-server/src/main/resources/html/js/spark-jobserver-ui.js <https://github.com/spark-jobserver/spark-jobserver/pull/361#discussion_r50154555>:
> 
> > @@ -10,7 +10,7 @@ function getJobs() {
> >        $.each(jobs, function(key, job) {
> >          var items = [];
> >          items.push("<tr>");
> > -        items.push("<td>" + job.jobId + "</td>");
> > +        items.push("<td><a href='./jobs/" + job.jobId + "'>" + job.jobId + "</a> (<a href='./jobs/" + job.jobId + "/config'>C</a>)</td>");
> I defaulted to C since a wall of Config seemed overwhelming to me. But I can change it, please let me know either way.
> 
>  <https://cloud.githubusercontent.com/assets/181554/12428457/78d9c990-be99-11e5-981d-6c059471cc17.png>
> —
> Reply to this email directly or view it on GitHub <https://github.com/spark-jobserver/spark-jobserver/pull/361/files#r50154555>.
>